client/public/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
```

Comment:
Add a favicon, It would look much better. Use Figma for designing a simple logo.